terraso_backend/apps/project_management/models/projects.py

```diff
@@ -78,6 +78,7 @@ class Meta(BaseModel.Meta):
     )
 
     seen_by = models.ManyToManyField(User, related_name="+")
+    site_instructions = models.TextField(null=True, blank=True)
```

Comment:
question: what is the difference between site_note and site_instructions?

Reply:
Many people can attach Site Notes to a site; several different data collectors might use them to make observations that aren't otherwise available in the data inputs ("I had to move because there was a tractor", "The ground was frozen", "Watch out for the goats"). 

Site Instructions, on the other hand, is a single field set on the Project that will display at the top of the Notes screen for each Site in the Project.